src/Data/VectorSpace/Core.agda

```diff
@@ -0,0 +1,94 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Abstract vector spaces.
+--
+-- A "vector space" is a Module with a commutative, homomorphic inner
+-- product and a complete set of building blocks for mapping the space.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.VectorSpace.Core where
```

Comment:
This shouldn't live in `Data`. If I had to make a stab at it, a better place would be to create a new subfolder `Algebra.Linear` for linear algebra and place `VectorSpace` in `Algebra.Linear.Bundles`?

Reply:
**==> Code moved, as requested, in next push.**